traitsui/tests/editors/test_code_editor.py

```diff
@@ -60,12 +61,10 @@ def is_line_numbers_visible(ui):
             return txt_ctrl.line_number_widget.isVisible()
 
         def test_line_numbers_visibility(show=True):
-            with store_exceptions_on_all_threads():
-                code_model = CodeModel()
-                code_view = CodeView(model=code_model, show_line_numbers=show)
-                ui = code_view.edit_traits()
+            code_model = CodeModel()
+            code_view = CodeView(model=code_model, show_line_numbers=show)
+            with store_exceptions_on_all_threads(), create_ui(code_view) as ui:
                 self.assertEqual(is_line_numbers_visible(ui), show)
-                ui.control.close()
```

Comment:
I don't think this action is performed in `ui.dispose()` so it might be better to leave it in the test for now (same in the test below).